mlir/include/mlir/Dialect/LLVMIR/LLVMOps.td

```diff
@@ -1087,6 +1087,7 @@ def LLVM_GlobalOp : LLVM_Op<"mlir.global",
     OptionalAttr<UnnamedAddr>:$unnamed_addr,
     OptionalAttr<StrAttr>:$section,
     OptionalAttr<SymbolRefAttr>:$comdat,
+    DefaultValuedAttr<LLVM_DIGlobalVariableExpressionAttr, "::mlir::LLVM::DIGlobalVariableExpressionAttr()">:$dbg_expr,
```

Comment:
```suggestion
    DefaultValuedAttr<LLVM_DIGlobalVariableExpressionAttr, "{}">:$dbg_expr,
```
Would {} here as well? If not, otherwise I would indent the second line with two spaces more than the line above.